sessions/src/main/kotlin/io/spine/examples/pingh/sessions/UserSessionProcess.kt

```diff
@@ -117,3 +152,8 @@ internal class UserSessionProcess :
         this.authenticationService = authenticationService
     }
 }
+
+/**
+ * Compares this `Timestamp` with the passed one.
+ */
+private operator fun Timestamp.compareTo(other: Timestamp): Int = Timestamps.compare(this, other)
```

Comment:
Let's make this a common extension function. I think, we already have `TimestampExts` somewhere.